admin/classes/domain/IssueLog.php

```diff
@@ -23,7 +23,38 @@ protected function defineRelationships() {}
 
 	protected function overridePrimaryKeyName() {}
 
+  public function getTypeShortName() {
+     $query = "SELECT shortName 
+            FROM IssueLogType
+            WHERE issueLogTypeID = '" . $this->issueLogTypeID . "'";
+    $result = $this->db->processQuery($query, 'assoc');
+    return $result['shortName'];
+ 
+  }
+
+  public function allExpandedAsArray($organizationID) {
+
+    $query = "SELECT IssueLog.*, Organization.name, IssueLogType.shortName from IssueLog
+          LEFT JOIN IssueLogType ON IssueLogType.issueLogTypeID = IssueLog.issueLogTypeID
+          LEFT JOIN Organization ON Organization.organizationID = IssueLog.organizationID";
+
+    if ($organizationID) {
+      $query .= " WHERE Organization.OrganizationID = $organizationID";
```

Comment:
When including parameters in SQL queries, we should sanitize the input using something like mysqli_real_escape_string().